paasta_tools/check_kubernetes_services_replication.py

```diff
@@ -89,9 +66,9 @@ def check_healthy_kubernetes_tasks_for_service_instance(
     )
 
 
-def check_service_replication(
+def check_kubernetes_service_replication(
```

Comment:
I think using the word service at all is kind of confusing (just because of the clash with the kubernetes service concept vs yelp's). how about `check_kubernetes_pod_replication` ?

Reply:
done